x/cert/keeper/key.go

```diff
@@ -5,21 +5,20 @@ import (
 	"math/big"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
 
 	"github.com/ovrclk/akash/x/cert/types"
 )
 
 const (
-	keyAddrPrefixLen = 1 + sdk.AddrLen
-)
-
-var (
-	prefixCertificateID = []byte{0x01}
+	keyAddrPrefixLen = 1 /*len(PrefixCertificateID)*/ + 1 /*owner_address_len (1 byte)*/
```

Comment:
something like that to make it more readable?
```
keyAddrPrefixLen = 2 // 1 byte for PrefixCertificateID) followed by owner_address_len byte
```

Reply:
It is `const`. The diff shows `var` on the LHS, it is already `const` on the RHS.